docs/docs/components/text.md

```diff
@@ -47,6 +47,10 @@ numberOfLines: number = 0;
 // Is the text selectable (affects mouse pointer and copy command)
 selectable: boolean = false;
 
+// Mouse & Touch Events
+onPress?: (e: SyntheticEvent) => void;
```

Comment:
Both of these have default values of "undefined". Please add "= undefined" at the end to indicate so. (The value after the "=" in the documentation indicates the default value.)